backend/oracleobjectstorage/options.go

```diff
@@ -225,16 +221,18 @@ The minimum is 0 and the maximum is 5 GiB.`,
 		Advanced: true,
 	}, {
 		Name:     "memory_pool_flush_time",
-		Default:  memoryPoolFlushTime,
+		Default:  fs.Duration(time.Minute),
+		Hide:     fs.OptionHideBoth,
 		Advanced: true,
 		Help: `How often internal memory buffer pools will be flushed.
 
 Uploads which requires additional buffers (f.e multipart) will use memory pool for allocations.
 This option controls how often unused buffers will be removed from the pool.`,
 	}, {
 		Name:     "memory_pool_use_mmap",
```

Comment:
Why doesn't this  rclone's flag `--use-mmap` which already exists?

Reply:
It will after this is merged :-)